AutoCollection/Performance.ts

```diff
@@ -208,10 +163,6 @@ class AutoCollectPerformance {
             this._client.trackMetric(combinedName + "nice", totalNice / combinedTotal);
```

Comment:
Had offline discussion. Will keep the clean trackMetric names and ensure compatibility is kept on backend for legacy name conversion